BunqSdk/Exception/ForbiddenException.cs

```diff
@@ -2,7 +2,8 @@
 {
     public class ForbiddenException : ApiException
     {
-        public ForbiddenException(int responseCode, string message) : base(responseCode, message)
+        public ForbiddenException(int responseCode, string message, string responseId) :
```

Comment:
Why declare a ctor explicitly when it just calls its parent without doing anything? *Same for other exception types*

Reply:
https://stackoverflow.com/a/7689774